add-new-bcd.js

```diff
@@ -123,7 +122,7 @@ const main = async () => {
   console.log('Done!');
 };
 
-main().catch((e) => {
-  console.error(e);
-  process.exit(1);
-});
+/* istanbul ignore if */
+if (esMain(import.meta)) {
+  main();
```

Comment:
Don't we still need to catch promise rejection here?

Reply:
One nice feature ESM offers is top-level async, so we don't have to worry about writing a `catch()` block.  Admittedly though, I wasn't using it correctly, as one still needs to `await` the function.  Fixed!

Reply:
Oh, I'm glad I asked then :)